Keas.Mvc/ClientApp/components/Equipment/EquipmentBigFixSearchId.tsx

```diff
@@ -0,0 +1,280 @@
+import * as PropTypes from "prop-types";
+
+import * as React from "react";
+
+import {
+    Button,
+    Form,
+    FormFeedback,
+    FormGroup,
+    Input,
+    Label,
+    Modal,
+    ModalBody,
+    ModalFooter,
+    Table
+} from "reactstrap";
+import { IBigFixSearchedName } from "../../Types";
+
+interface IProps {
+    addBigFixId: (property: string, id: string) => void;
+}
+
+interface IState {
+    isFetched: boolean;
+    isSearching: boolean;
+    isValidSearch: boolean;
+    searchModal: boolean;
+    selectedFeild: string;
+    valueToBeSearched: string;
+    listOfComputers: IBigFixSearchedName[];
+}
+
+export default class EquipmentBigFixInfo extends React.Component<IProps, IState> {
+    public static contextTypes = {
+        fetch: PropTypes.func,
+        permissions: PropTypes.array,
+        router: PropTypes.object,
+        team: PropTypes.object
+    };
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            isFetched: false,
+            isSearching: false,
+            isValidSearch: true,
+            listOfComputers: [],
+            searchModal: false,
+            selectedFeild: "Name",
+            valueToBeSearched: ""
+        };
+    }
+
+    public render() {
+        return (
+            <>
```

Comment:
Empty tag

Reply:
its purpose is to combine a set of elements into one anonymous tag so that the render method can return it.

Reply:
@NabilFurmoli why not just use a div?

Reply:
ok I will update it